services/QuillLMS/config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,10 @@
+SecureHeaders::Configuration.default do |config|
+  config.csp = SecureHeaders::OPT_OUT
+  config.cookies = {
```

Comment:
The cookie configuration for `SameSite=lax`,`secure`, and `httponly` might already be set out of the box: https://github.com/github/secure_headers/blob/main/docs/upgrading-to-5-0.md